.circleci/config.yml

```diff
@@ -31,7 +31,3 @@ jobs:
       - run:
           name: Upload code coverage report
           command: ./node_modules/.bin/codecov
-
-      - run:
-          name: Upload code coverage report - CoverAlls
-          command: ./node_modules/.bin/coveralls < ./coverage/lcov.info
```

Comment:
Was this an accidental inclusion?

Reply:
No.

Reply:
Looks like we are temporarily removing this and fixing CI after.